horovod/run/task_fn.py

```diff
@@ -55,14 +56,15 @@ def _task_fn(index, driver_addresses, num_hosts, tmout, key):
 if __name__ == '__main__':
     if len(sys.argv) != 6:
         print(
-                'Usage: %s <index> <service addresses> <num_hosts> <tmout> <key>' %
+                'Usage: %s <index> <service addresses> <num_hosts> <tmout> '
+                '<settings>' %
                 sys.argv[0])
         sys.exit(1)
 
     index = codec.loads_base64(sys.argv[1])
     driver_addresses = codec.loads_base64(sys.argv[2])
     num_hosts = codec.loads_base64(sys.argv[3])
     tmout = codec.loads_base64(sys.argv[4])
-    key = codec.loads_base64(sys.argv[5])
+    settings = codec.loads_base64(sys.argv[6])
```

Comment:
Should it be `sys.argv[5]`?  Did you test? :P

Reply:
fixed. was waiting for the tests to run in CI.